cmd/eval.go

```diff
@@ -248,6 +273,7 @@ Loads a single JSON file, applying it to the input document; or all the schema f
 	evalCommand.Flags().BoolVarP(&params.failDefined, "fail-defined", "", false, "exits with non-zero exit code on defined/non-empty result and errors")
 
 	// Shared flags
+	addCapabilitiesFlag(evalCommand.Flags(), params.capabilities)
```

Comment:
A fun side effect of this is that we can now use `opa eval` to _eval like it's 20202_:
```
$ opa eval -fpretty --capabilities capabilities/v0.17.0.json 'base64.is_valid("foo")'
1 error occurred: 1:1: rego_type_error: undefined function base64.is_valid
$ opa eval -fpretty --capabilities capabilities.json 'base64.is_valid("foo")'
false
```